lumigator/python/mzai/backend/backend/api/routes/jobs.py

```diff
@@ -58,12 +56,16 @@ def create_annotation_job(
     reference model should be used to generate annotations.
     See more: https://blog.mozilla.ai/lets-build-an-app-for-evaluating-llms/
     """
-    inference_job_create_request = JobInferenceCreate(
-        **job_create_request.dict(),
+    inference_job_create_config = JobAnnotateConfig(
+        **job_create_request.job_config.dict(),
         model="hf://facebook/bart-large-cnn",
         output_field="ground_truth",
     )
-    inference_job_create_request.store_to_dataset = True
+    inference_job_create_config.store_to_dataset = True
```

Comment:
Should this be moved inside the JobAnnotateConfig constructor one line abouve?

Reply:
If we make the distinction of request and config as whether it's handled by the backend or the job, I'd keep it where it is. The storage of results into a dataset is actually the a for the generic backend-job interaction and not something to be handled in the job itself.